documentation/docs/steps/abapEnvironmentPushATCSystemConfig.md

```diff
@@ -0,0 +1,139 @@
+
+```yaml
+steps:
+  abapEnvironmentPushATCSystemConfig:
+    abapCredentialsId: 'abapCredentialsId',
+    host: 'https://myABAPendpoint.com',
+    atcSystemConfigFilePath: 'atcSystemConfig.json',
+```
+
+To trigger a create/update ATC System Configuration step an ATC System configuration file `atcSystemConfig.json` will be needed. Check section 'ATC System Configuration file example' for more information.
+
+### Create/Update an ATC System Configuration via Cloud Foundry Service Key example in Jenkinsfile
+
+The following example triggers a Create/Update operation on an ATC System Configuration via reading the Service Key of an ABAP instance in Cloud Foundry.
+
+You can store the credentials in Jenkins and use the cfCredentialsId parameter to authenticate to Cloud Foundry.
+The username and password to authenticate to ABAP system will then be read from the Cloud Foundry service key that is bound to the ABAP instance.
+
+This can be done accordingly:
+
+```groovy
+abapEnvironmentPushATCSystemConfig(
+    cfApiEndpoint : 'https://test.server.com',
+    cfOrg : 'cfOrg',
+    cfSpace: 'cfSpace',
+    cfServiceInstance: 'myServiceInstance',
+    cfServiceKeyName: 'myServiceKey',
+    abapCredentialsId: 'cfCredentialsId',
+    atcSystemConfigFilePath: 'atcSystemConfig.json',
+    script: this,
+)
+```
+
+### Create/Update an ATC System Configuration via direct ABAP endpoint configuration in Jenkinsfile
+
+This example triggers a Create/Update operation on an ATC System Configuration run directly on the ABAP endpoint.
+
+In order to trigger the Create/Update operation on an ATC System Configuration you have to pass the username and password for authentication to the ABAP endpoint via parameters as well as the ABAP endpoint/host. You can store the credentials in Jenkins and use the abapCredentialsId parameter to authenticate to the ABAP endpoint/host.
+
+This must be configured as following:
+
+```groovy
+abapEnvironmentPushATCSystemConfig(
+    abapCredentialsId: 'abapCredentialsId',
+    host: 'https://myABAPendpoint.com',
+    atcSystemConfigFilePath: 'atcSystemConfig.json',
+    script: this,
+)
+```
+
+To Create/Update an ATC System Configuration a file `atcSystemConfig.json` will be needed. Check section 'ATC System configuration file example' for more information.
+
+### ATC System configuration file example
+
+The step always performs a check first, if an ATC System Configuration with the same name provided in the file `atcSystemConfig.json` with the attribute conf_name.
+This file contains an JSON Representation of an ATC System Configuration. Some json file examples can be found below.
+In case an ATC System Configuration with this name already exists, by default, the step would perform an Update of this ATC System Configuration with the ATC System Configuration information provided in file `atcSystemConfig.json`.
```

Comment:
```suggestion
In case an ATC System Configuration with this name already exists, by default, the step would perform an update of this ATC System Configuration with the ATC System Configuration information provided in file `atcSystemConfig.json`.
```